Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectNavigation.cs

```diff
@@ -185,6 +180,48 @@ public override TokenParser ProvisionObjects(Web web, ProvisioningTemplate templ
 
         #region Utility methods
 
+        private bool WebSupportsProvisionNavigation(Web web, ProvisioningTemplate template)
+        {
+            if (template.Navigation == null) return true;
+
+            bool isNavSupported = true;
+            // The Navigation handler for managed metedata only works for sites with Publishing Features enabled
+            if (!web.IsPublishingWeb())
+            {
+                if (template.Navigation.GlobalNavigation != null
+                    && template.Navigation.GlobalNavigation.NavigationType == GlobalNavigationType.Managed)
+                {
```

Comment:
Ok, about this one ... it is a very very edge case. I think we can skip it, so far and see if it will hurt someone ...